src/server/index.tsx

```diff
@@ -26,7 +26,28 @@ server.use(express.json() as RequestHandler);
 server.use(express.urlencoded({ extended: false }) as RequestHandler);
 server.use("/static", express.static(path.resolve("./dist")));
 server.use("/docs", express.static(path.resolve("./dist/assets/docs")));
-server.use("/api", express.static(path.resolve("./dist/assets/api")));
+
+// The v0.19 release docs
+// TODO OpenAPI isn't currently working for the latest docs. This can be changed after the next release.
+// server.use("/api/v0.19", express.static(path.resolve("./dist/assets/api_v0.19.html")));
```

Comment:
Its fine if openapi docs only work for the new release. We dont need to put unnecessary effort into the old version, and it gives another reason for devs to upgrade. So you can remove those comments.